### PR TITLE
Intro

### PR DESCRIPTION
**Pull request by tomsbzbrs**
I'm sorry for the previous pushes/requests, I had a few problems that surfaced only after issuing the pull request. This is the final version.

### CODE REVIEW

File: Exercises/Basics/AddNumbers-01/Program.cs [hunk 1]
```diff
@@ -15,7 +11,10 @@ static void Main(string[] args)
             Console.ReadKey();
         }
 
-        static int AddNumbers(int a int b int c)
-            return a + b + c
+        static int AddNumbers(int a, int b, int c)
```
Comment (ausmoons) [inline Exercises/Basics/AddNumbers-01/Program.cs:14]: Tev kkas nav ar gitignore. Te ir .vs faili un man lokāli arī pievienojas lieki faili

File: Exercises/Basics/AddNumbers-01/Program.cs [hunk 2]
```diff
@@ -15,7 +11,10 @@ static void Main(string[] args)
...
+        {
+            return a + b + c;
+        }
+
```
Comment (ausmoons) [inline Exercises/Basics/AddNumbers-01/Program.cs:18]: Lieka tukša rinda

**Review by ausmoons — CHANGES_REQUESTED** [review]

File: Exercises/Basics/AddNumbers-01/Program.cs (hunk 1 above)
Comment (tomsbzbrs) [reply Exercises/Basics/AddNumbers-01/Program.cs:14]: ok, meklēju

**Review by ausmoons — APPROVED** [review]
Sakārto gitignore mainā

File: Exercises/Basics/AddNumbers-01/Program.cs (hunk 2 above)
Comment (tomsbzbrs) [reply Exercises/Basics/AddNumbers-01/Program.cs]: izlabots

**Pull request merged**